stable/spark-operator/values.yaml

```diff
@@ -99,12 +99,8 @@ webhook:
   namespaceSelector: ""
   # -- The annotations applied to init job, required to restore certs deleted by the cleanup job during upgrade
   initAnnotations:
-    "helm.sh/hook": pre-install, pre-upgrade
-    "helm.sh/hook-weight": "50"
   # -- The annotations applied to the cleanup job, required for helm lifecycle hooks
   cleanupAnnotations:
-    "helm.sh/hook": pre-delete, pre-upgrade
-    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
imo leave here, next time we will fork the spark operator helm chart, it won't conflict or raise a question

Reply:
Done